content/blog/2020-02-10-introduction-to-machine-learning-with-the-tidyverse/index.Rmarkdown

```diff
@@ -0,0 +1,191 @@
+
+```{r echo=FALSE, out.width="49%", fig.link="https://github.com/allisonhorst/stats-illustrations", fig.cap="Art by Allison Horst", fig.align='center'}
+knitr::include_graphics("recipes.png")
+```
+
+## Is this course for me?
+
+Before workshops for this year's conf were announced, we framed two questions to help potential learners gauge whether this workshop was the right one for them:
+
+---
+
++ Can you use mutate and purrr to transform a data frame that contains list columns?
+
++ Can you use the ggplot2 package to make a large variety of graphs?
+
+If you answered "no" to either question, you can brush up on these topics by working through the online tutorials at https://rstudio.cloud/learn/primers.
+
+---
+
+These questions were driven by the fact that when we started developing the workshop, tidymodels code heavily relied on the purrr package (see an end-to-end code example from Max's Applied Machine Learning workshop at rstudio::conf(2019) [here](https://github.com/topepo/rstudio-conf-2019/blob/2aaf4c24cd90cfa91cf2cdde256d68341f21133b/Materials/Part_2_Basic_Principles.R#L137-L212)). However, between the time we first conceived of the workshop and when we taught it, a lot of the tidymodels API had changed (for the better!). In hindsight, here is the minimum tidyverse comfort level you need to be able to dive into tidymodels:
+
++ Can you use the [pipe operator](https://r4ds.had.co.nz/transform.html#combining-multiple-operations-with-the-pipe) to combine a sequence of functions to transform objects in R (like a data frame)?
+
++ Can you work with tibbles (or data frames) that contain [list columns](https://r4ds.had.co.nz/many-models.html#list-columns-1)?
+
++ Can you use [`dplyr::select()` helper functions](https://tidyselect.r-lib.org/reference/select_helpers.html)? (this helps a lot when you get to the [recipes][recipes] package)
+
+Do you have to be a tidyverse enthusiast to use tidymodels? No, having a comfort level with the pipe, tibbles, and list columns helps.
+
+In the rest of this post, I'll walk you through the materials available through the [workshop website](https://conf20-intro-ml.netlify.com/):
+
+```{r echo=FALSE, out.width = '100%'}
+knitr::include_url("https://conf20-intro-ml.netlify.com/")
+```
+
+
+## Packages
+
+For the code, we obviously made heavy use of packages from [tidymodels](https://tidymodels.github.io/tidymodels/), which is available (like the [tidyverse](https://tidyverse.tidyverse.org/)) as a meta-package that bundles most of the building blocks we needed:
```

Comment:
"The code made heavy use of..." rather than "For the code, we obviously made heavy use of..."